qutebrowser/completion/models/miscmodels.py

```diff
@@ -60,17 +60,31 @@ def helptopic():
 
 def quickmark():
     """A CompletionModel filled with all quickmarks."""
+    def delete(data):
```

Comment:
pylint wants a docstring here - I probably should add an option to pylint to not require that for functions defined inside functions :wink: